README.md

```diff
@@ -47,4 +47,5 @@ Any of these can be passed into the options object for each function.
 | numbers                  | Boolean, put numbers in password.           |     false     |
 | symbols                  | Boolean, put symbols in password.           |     false     |
 | uppercase                | Boolean, use uppercase letters in password. |      true     |
-| excludeSimilarCharacters | Excludes similar chars, like 'i' and 'l'.   |     false     |
\ No newline at end of file
+| excludeSimilarCharacters | Excludes similar chars, like 'i' and 'l'.   |     false     |
+| strict                   | Generated password must obey specified rules|     false     |
```

Comment:
Updated to: "Generated password must include at least one character from each pool."